### PR TITLE
Update StartScreen.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue - "fill in the blank" assignments - Description has been updated now.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by get-itips** [comment]
Once the typo that Dario mentions is fixed, it is ok to be merged by me.

**Comment by tseward** [comment]
Agreed looks good -- just needs to clarify the dash.

File: docset/winserver2012r2-ps/startscreen/StartScreen.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ ms.reviewer: brianlic
 
 # StartScreen Module
 ## Description
-{{Manually Enter Description Here}}
+This reference provides cmdlet descriptions and syntax for all Start Screen -specific cmdlets. It lists the cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/startscreen/StartScreen.md]:
```suggestion
This reference provides cmdlet descriptions and syntax for all Start Screen-specific cmdlets. It lists the cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```

**Comment by sowjanya365** [comment]
> The middle dash before the word "specific" is ok or it's a typo?

Hi @dariomws306

I've added the dash to the sentence after referring to the other articles with the same kind of description. So I just started following the same format. Hope this clarifies. Let me know if you think we need to omit the dash. Thank you! :)

"This reference provides cmdlet descriptions and syntax for all Server Manager-specific cmdlets"

**Comment by sowjanya365** [comment]
> Yes, it's true @sowjanya365, sorry about that. I see the dash in other similar articles after I left this comment. So I think it's ok. Thanks!

Thank you, @dariomws306 :)

**Pull request merged**